modules/dev-tools/deprecation.php

```diff
@@ -123,7 +123,7 @@ public function parse_version( $version ) {
 		$version_explode_count = count( $version_explode );
 
 		if ( $version_explode_count < 3 || $version_explode_count > 4 ) {
-			trigger_error( 'Invalid Semantic Version string provided' );
+			trigger_error( 'Invalid Semantic Version string provided' . $version );
```

Comment:
It's better to `dump` it in case it's null or empty string.
```suggestion
			trigger_error( 'Invalid Semantic Version string provided' . var_export( $version, 1 ) );
```